models/gif_psc_exp.cpp

```diff
@@ -292,8 +292,10 @@ nest::gif_psc_exp::calibrate()
   const double tau_m = P_.c_m_ / P_.g_L_;
 
   // these are determined according to a numeric stability criterion
-  V_.P21ex_ = propagator_32( P_.tau_ex_, tau_m, P_.c_m_, h );
-  V_.P21in_ = propagator_32( P_.tau_in_, tau_m, P_.c_m_, h );
+  PropagatorExp prop_ex( P_.tau_ex_, tau_m, P_.c_m_ );
+  PropagatorExp prop_in( P_.tau_in_, tau_m, P_.c_m_ );
+  V_.P21ex_ = prop_ex.evaluate( h );
+  V_.P21in_ = prop_in.evaluate( h );
```

Comment:
For all models which do not need the propagator object elsewhere, I'd write directly
```suggestion
  V_.P21ex_ = PropagatorExp( P_.tau_ex_, tau_m, P_.c_m_ ).evaluate( h );
  V_.P21in_ = PropagatorExp( P_.tau_in_, tau_m, P_.c_m_ ).evaluate( h );
```